google-account-plugin/src/com/google/cloud/tools/intellij/login/CredentialedUser.java

```diff
@@ -49,19 +51,12 @@ protected CredentialedUser(String email) {
    * Creates a credentialed user.
    */
   public CredentialedUser(GoogleLoginState state,
-      final IGoogleLoginCompletedCallback updateUserCallback) {
+      @Nullable final IGoogleLoginCompletedCallback callback) {
```

Comment:
the callback name should include what event it handles.  I guess in this case it should be something like `onLoginComplete`.